packages/contracts/src/diamond/token/ERC1155UbiquityForDiamond.sol

```diff
@@ -0,0 +1,184 @@
+// SPDX-License-Identifier: MIT
```

Comment:
ERC1155UbiquityForDiamond  --- The name doesn't make fully sense to me. Why couldn't we name ERC1155Ubiquity? lol

Reply:
It is because  ERC1155Ubiquity already exist in the current implementation but yes once we delete the non diamond contracts we should rename